gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java

```diff
@@ -368,7 +368,11 @@ public void onSuccess(Void ignore, HttpHeaders responseHeaders) {
 
           @Override
           public void onFailure(GoogleJsonError e, HttpHeaders responseHeaders) {
-            deletes.put(tuple.x(), Tuple.<Boolean, StorageException>of(null, translate(e)));
+            if (e.getCode() == 404) {
```

Comment:
Sure I'll move all the `NOT_FOUND` checks to `DefaultStorageRpc`.

I am fine with documenting the "return null" case in `DefaultStorageRpc`, do you think we should do that in javadoc format or simple line/block comments to avoid adding javadoc to what we wanted to leave undocumented?